homework_2/offerfeed/internal/infrastructure/http/notAJsonFeed.go

```diff
@@ -0,0 +1,94 @@
+package http
+
+import (
+	"context"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"code-cadets-2021/homework_2/offerfeed/internal/domain/models"
+)
+
+const notAJsonFeedURL = "http://18.193.121.232/axilis-feed-2"
+
+type NotAJsonFeed struct {
+	updates    chan models.Odd
+	httpClient *http.Client
+}
```

Comment:
Is it okay to name it something like AxilisOfferFeed2 or AnotherAxilisOfferFeed? My imagination in naming variables has always been concerning.  :smile:

Reply:
Yes, those names are great. Usually we would want to avoid numbers in names since that is usually a [code smell](https://en.wikipedia.org/wiki/Code_smell) for lack of domain exploration, but in this case the URL contains it in the name, so it would be no problem.